### DISCUSSION TITLE
How to create new pages in this boilerplate ?

### DISCUSSION THREAD

**Discussion by harshalbhangale** [agustinusnathaniel/nextarter-chakra#145] [Q&A]
How to create new pages in this boilerplate ?  I have a locale setup for different languages , but does not able to make a new page ! Please help

**Comment by agustinusnathaniel** [comment]
Hi @HarshalBhangale you can add new page in `src/app` folder

References:
https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#pages

**Reply by harshalbhangale** [reply]
Got it thanks.. actually, in our repo it was a locale setup for multi language. And I was facing problems with multiware. 

Anyways Thanks for the reply.
Appreciate it.

**Reply by agustinusnathaniel** [reply]
Do you meant to implement i18n? I'd suggest to check out https://next-intl-docs.vercel.app/
